app/src/main/java/io/multy/ui/fragments/dialogs/PrivateKeyDialogFragment.java

```diff
@@ -97,8 +97,9 @@ private String getPrivateKey() {
             byte[] seed = RealmManager.getSettingsDao().getSeed().getSeed();
             int walletIndex = viewModel.getWalletLive().getValue().getWalletIndex();
             int addressIndex = address.getIndex();
-            int currency = NativeDataHelper.Currency.BTC.getValue();
-            return NativeDataHelper.getMyPrivateKey(seed, walletIndex, addressIndex, currency);
+            return NativeDataHelper.getMyPrivateKey(seed, walletIndex, addressIndex,
+                    NativeDataHelper.Blockchain.BLOCKCHAIN_BITCOIN.getValue(),
+                    NativeDataHelper.BlockchainNetType.BLOCKCHAIN_NET_TYPE_TESTNET.getValue());
         } catch (Throwable throwable) {
             throwable.printStackTrace();
             Toast.makeText(Multy.getContext(), "Error while build private key ;(", Toast.LENGTH_SHORT).show();
```

Comment:
Get the normal error text from @vadimicus. Don't show user any smiles or other selfmade text.